test/api/kafka.js

```diff
@@ -29,6 +29,33 @@ describe("Kafka", function() {
             assert.strictEqual(connectSpy.called, true);
             assert.strictEqual(sendSpy.called, true);
 
+            clientStub.restore();
+        });
+        it("should be able to trigger a message in the adapter for topics with global diffusion", async () => {
+            const bus = new api.API({ hosts: "localhost:9092", adapter: "kafka" });
+
+            const fakeProducer = {
+                connect: () => {},
+                send: () => {}
+            };

```

Comment:
Can we make this fake producer "send" method actually send the event and then store it somewhere so that we can make asserts on it? This would have caught the previous error for payload missformat.